### DISCUSSION TITLE
Missing BackgroundColor with 8.5.0

### DISCUSSION THREAD

**Discussion by ipoweb** [dlemstra/Magick.NET#1098] [Help]
Hello,
i update Magick.net from 7.15.3 to 8.5.0. 

Actually i use interface ImageMagick for my image to define BackgroundColor Property but with the last version, this property doesn't exist on interface. I don't know why, and i need to know if it's depreciated property or not or is it bug, and how i can fix this error ?

```
image.Extent(size, g, image.BackgroundColor);
MagickColor forcedBgColor = image.BackgroundColor == MagickColors.Transparent ? MagickColors.White : image.BackgroundColor;
MagickColor bgColor = addBackground ? forcedBgColor : image.BackgroundColor;
```

Other break change i don't understand, in past i used this, now MagickImageCollection required ushort item instead for image when i add to images collection

`
using (MagickImageCollection images = new MagickImageCollection())
 images.Add(image);
`

Thanks for your help

**Comment by dlemstra** [comment]
Between those versions a new interface was introduced. This interface `IMagickImage<TQuantumType>` is different depending on the Magick.NET version that you are using. This was done to make it possible to create the new Magick.NET.Core library. It looks like you will need to use `IMagickImage<ushort>` where you were using `IMagickImage`.

**Comment by ipoweb** [comment]
Thanks a lot for your quick answer. You save my life :)
it works ! Have Happy New Year !
